README.md

```diff
@@ -30,6 +30,14 @@ We use a tool called migrate-mongo: https://www.npmjs.com/package/migrate-mongo
   - With this command, migrate-mongo will revert (only) the last applied migration
 - status: `npx migrate-mongo status`
 
+## Extending stub-schema for running DB migration Tests
+
+- extend upon the [schema builder](/test/integration/submission/migration_utils/schema_builder.ts) with the new schema
+- run the script using `npx ts-node schema_builder.ts` to generate the [migration-stub-schema.json](/test/integration/submission/migration-stub-schema.json)
```

Comment:
why is this not all done in memory ?

Reply:
this can be done before all the tests? isnt that how you had it before ?

Reply:
I changed it because it's still not purely automatic anyway; i.e. after making the schema, the diffs part of it needs to be brought in from lectern's diff calculator, and added to the file. so it's not reliable to be generated on the fly and we thought it would be better to just avoid the expensive computation before every test.

Reply:
it's fair that it's not automatic, yeah the diffs part is brutal and can be deceiving if lectern changed it's diff outputs somehow, our tests will pass when they shouldn't

Reply:
I'll create a ticket to run tests against a real lectern container